packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -133,6 +133,13 @@
     background-color: transparentize(cv("gray", "100"), 0.6);
     color: cv("gray", "600");
   }
+
+  .icon {
+    display: block;
+    margin-block-start: 0.0625em;
+    margin-block-end: -0.0625em;
+    line-height: 0.9;
+  }
```

Comment:
looking forward to having some of these styles inline with Icon (more than likely via Box) to reduce duplication like this